HumanControls/HumanControls.cpp

```diff
@@ -145,4 +149,21 @@ void HumanControls::connect()
 {
     //Connect to the robot
     this->m_isConnected = true;
+    m_state.root()["hCtrl"]["conn"] = "Active";
+}
+
+void HumanControls::setError(const char *format, ...)
+{
+    va_list args;
+    char message[32];
+    va_start(args, format);
+    vsprintf(message, format, args);
+    va_end(args);
+
+    m_state.root()["err"] = message;
```

Comment:
Let's not set the controls error on the robot.

Reply:
Done.